docs/source/creating_databases.rst

```diff
@@ -0,0 +1,91 @@
+Creating the databases
+======================
+
+First, make sure you have Python and Postgres installed:
+
+-  :doc:`installing_python`
+-  :doc:`installing_postgres`
+
+
+Creating the databases
+----------------------
+
+OSX
+~~~
+
+After installing Postgres, you will need to create two databases:
+one for your experiments to use, and a second to support importing saved
+experiments. It is recommended that you also create a database user.
+
+Naviagate to a terminal and type:
+::
+
+    createuser -P dallinger --createdb
+    (Password: dallinger)
+    createdb -O dallinger dallinger
+    createdb -O dallinger dallinger-import
```

Comment:
I *think* we may no longer need the `dallinger-import` db. @alecpm has an for verifying.

Reply:
happy to take it out, once verified.

Reply:
Agreed. and it doesn't hurt if they have an extra potentially not in use DB installed.